### DISCUSSION TITLE
Google Vignette (auto) ads issue with Next 13.4+ versions using app directory

### DISCUSSION THREAD

**Discussion by militiwari** [vercel/next.js#50423] [App Router]
I am using the following Script Tag in RootLayout

`<Script
          id="Adsense-id"
          data-ad-client="ca-pub-***************"
          async={true}
          strategy="beforeInteractive"
          src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"
        />`

and it works fine till nextjs 13.3 in app directory.

But in 13.4 onwards when I close the "Vignette ads" using the close button or cross symbol provided by Google, it redirects me to a page with code in text format (it does not render, raw code is visible).

I tried it till 13.4.4 and still this issue is there.

This is the example of Vignette ads with close button - https://ibb.co/qj2XcNh

Thanks

**Comment by militiwari** [comment]
Screenshot of the error -
https://ibb.co/D5ZFYG8

**Comment by shriekdj** [comment]
screenshot is not available there. but i can say this might help you, try this props to you Script Component. `strategy="lazyOnload"
          crossOrigin="anonymous"` this might help you it helped me. start vignette ads clearly.
